src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -738,6 +738,11 @@ public ArticlePageObject openRandomArticle(String wikiURL) {
     getUrl(wikiURL + URLsContent.SPECIAL_RANDOM);
     return new ArticlePageObject(driver);
   }
+  
+  public ArticlePageObject openMainPage(String wikiURL) {
+      getUrl(wikiURL);
+      return new ArticlePageObject(driver);
```

Comment:
different, but we don't have MainpagePageObject
We have two ArticlePageObjects instead

I would say MainpagePageObject extends ArticlePageObject. If I get your agreement for that solution @ludwikkazmierczak I will create MainpagePageObject as subclass of ArticlePageObject